doc/docs/reference/core_fields.md

```diff
@@ -32,3 +32,6 @@ enum('one', '2', 'III')
 
 A field that stores it's value as a generic foreign key to an Opal LookupList
 or as the value in a CharField.
+
+By default this is case insensitive, pass in `case_sensitive=True` to make
```

Comment:
I'd like a code example below this and more of an explanation of what 'case [in]sensitive' means in this context?